app/services/facebook_authenticator.rb

```diff
@@ -12,10 +12,14 @@ def initialize(auth_code)
   end
 
   def authenticate!
+    access_token  = get_access_token(@auth_code)
```

Comment:
we could remove initialization and just have `FacebookAuthenticator.authenticate!(auth_code)` to fix that, but seems like too much work for little gain.